packages/scene-composer/package.json

```diff
@@ -150,6 +150,7 @@
     "@react-three/postprocessing": "2.6.2",
     "@react-three/test-renderer": "^8.1.3",
     "@tanstack/react-query": "^4.29.15",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
this new dependency doesn't seem to be used anywhere. please remove it if so

Reply:
We can use it in the future instead of fireEvent. This is better version of fireEvent.

Reply:
we can add it in the future when it's actually in use. I don't see a reason to add a library in advance unless you know you will use it in next change

Reply:
we should not be using firevent actually. It is out dated. We should be using this and replacing fireevent.